integrations/telegram/src/index.ts

```diff
@@ -149,19 +154,22 @@ const integration = new bp.Integration({
       throw new Error('Handler received an empty chat id')
     }
 
+    const userId = data.message.from?.id
+    const chatId = data.message.chat?.id
+
+    if (!userId) {
+      throw new Error('Handler received an empty from id')
```

Comment:
nitpick to make the error message clearer

```suggestion
      throw new Error('Handler received message with empty "from.id" value')
```